core/src/main/scala/com/evolutiongaming/kafka/flow/kafka/KafkaModule.scala

```diff
@@ -43,17 +37,18 @@ object KafkaModule {
       consumerMetrics <- ConsumerMetrics.of(registry)
       _producerOf      = RawProducerOf.apply1[F](producerMetrics(applicationId).some)
       _consumerOf      = RawConsumerOf.apply1[F](consumerMetrics(applicationId).some)
+
       _healthCheck <- {
-        implicit val randomIdOf        = RandomIdOf.uuid[F]
-        implicit val journalProducerOf = JournalProducerOf[F](_producerOf)
-        implicit val journalConsumerOf = JournalConsumerOf[F](_consumerOf)
+        implicit val randomIdOf = RandomIdOf.uuid[F]
+        implicit val consumerOf = _consumerOf
+        implicit val producerOf = _producerOf
+
         val commonConfig = config.common.copy(clientId = config.common.clientId.map(id => s"$id-HealthCheck"))
-        val healthCheck = KafkaHealthCheck.of[F](
-          config = KafkaHealthCheck.Config.default,
-          kafkaConfig = KafkaConfig(
-            ProducerConfig(common = commonConfig, saslSupport = config.saslSupport, sslSupport = config.sslSupport),
-            config.copy(common    = commonConfig)
-          )
+
+        val healthCheck = KafkaHealthCheck.of(
+          KafkaHealthCheck.Config.default,
+          ConsumerConfig(common = commonConfig),
```

Comment:
can we do it in the way it was before? i mean `config.copy(common = commonConfig)` cause otherwise it doesn't copy some important stuff like SSL config etc.